udata_front/theme/gouvfr/datagouv-components/src/api/schemas.ts

```diff
@@ -49,14 +47,10 @@ let catalogRequest: Promise<Array<RegisteredSchema>> | null = null;
  * Get Schema Catalog
  */
 export function getCatalog() {
-  if(!config.schema_catalog_url) {
-    catalogRequest = Promise.resolve([]);
-  }
   if (catalogRequest) {
     return catalogRequest;
   }
-  catalogRequest = axios.get<SchemaResponseData>(config.schema_catalog_url)
+  catalogRequest = api.get<SchemaResponseData>("datasets/schemas/")
```

Comment:
I think we can remove `schema_catalog_url` / `SCHEMA_CATALOG_URL` config, right?

Reply:
I removed the `schema_catalog_url` const and the check to `SCHEMA_CATALOG_URL` in metadata file. 

Do we want to remove it from udata-front settings too ? If we do it, it should be added to all our environments `cfg` because it's still used in udata with a default setting as `None`.

Reply:
I think we should remove it since udata-front doesn't use it! Our dev/demo/prod env have the value set properly in their cfg.
I don't know if front-kit env would be impacted though :)

Reply:
front-kit get the variable from the `.env` file at build time 👍🏻 
I'm adding a note to the PR description about the setting change for devs and I'll merge it.